apis/groups/v1alpha1/group_types.go

```diff
@@ -111,10 +111,14 @@ type GroupParameters struct {
 	// +optional
 	SubGroupCreationLevel *SubGroupCreationLevelValue `json:"subgroupCreationLevel,omitempty"`
 
-	// Disable email notifications.
+	// Disable email notifications. Deprecated, use emailsEnabled instead.
 	// +optional
 	EmailsDisabled *bool `json:"emailsDisabled,omitempty"`
```

Comment:
Deprecation marker should be in a new line (ref. https://go.dev/blog/godoc):

```suggestion
	// Disable email notifications.
	//
	// Deprecated: Use emailsEnabled instead.
	// +optional
	EmailsDisabled *bool `json:"emailsDisabled,omitempty"`
```